pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,432 @@
+---
+title: AsyncAPI Structure
+weight: 32
+---
+The structure of a AsyncAPI document can be defined as a specific format in which the document of the specification is to be written. An AsyncAPI document may be made up of a single document or be divided into multiple, connected parts at your discretion. In the latter case, Reference Objects are used. The structure of a AsyncAPI document has certain fields that need to be followed and implemented.
+Structure of a AsyncAPI document lays out the blueprint for you to write down your specification based on your application. 
+
+## Root Elements 
+Root elements of an AsyncAPI document provide an overview of the API's characteristics and behavior. These root elements collectively define the structure, metadata, channels, components and more of a AsyncAPI document. They provide a comprehensive overview of the API's characteristics and behavior.
```

Comment:
"the structure" in this context are the certain fields that shape the certain object or the properties in a AsyncAPI document.

Reply:
From https://www.asyncapi.com/docs/reference/specification/latest#A2SObject :

> This is the root document object for the API specification. It combines resource listing and API declaration together into one document.

I just feel the word `structure` doesn't fit well because we are talking about the resources/objects the whole document contain instead.

Reply:
So would you like to simply remove it?